website/extensions/yjs.md

```diff
@@ -0,0 +1,31 @@
+# Yjs
+
+[`Yjs`](https://github.com/yjs/yjs) is a CRDT framework with a powerful abstraction of shared data.
+
+By integrating yjs and prosekit, it is possible to create a document editor that allows multiple users to collaborate.
+
+> In this online example, in order to have a complete collaborative experience, you will need to apply for `appId` and `token` from [tiptap cloud](https://cloud.tiptap.dev/).
+
+<!-- @include: @/examples/yjs.md -->
+
+## Usage
```

Comment:
Since `yjs` and `y-prosemirror` are peer dependencies, let's add some docs about the installation step.